lib/LTO/LTO.cpp

```diff
@@ -0,0 +1,177 @@
+//===--- LTO.cpp - Swift LTO ----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/LTO/LTO.h"
+#include "swift/AST/ASTContext.h"
+#include "swift/AST/IRGenOptions.h"
+#include "swift/AST/IRGenRequests.h"
+#include "swift/ClangImporter/ClangImporter.h"
+#include "swift/ClangImporter/ClangImporterOptions.h"
+#include "swift/Frontend/InputFile.h"
+#include "swift/Frontend/ModuleInterfaceLoader.h"
+#include "swift/Frontend/PrintingDiagnosticConsumer.h"
+#include "swift/SIL/SILModule.h"
+#include "swift/Serialization/SerializedModuleLoader.h"
+#include "swift/Serialization/SerializedSILLoader.h"
+#include "swift/Serialization/Validation.h"
+#include "swift/Subsystems.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/Bitcode/BitcodeWriter.h"
+#include "llvm/IR/Module.h"
+
+namespace swift {
+
+namespace lto {
+
+using namespace llvm;
+
+class InputModule {
+public:
+  explicit InputModule(Identifier ModuleName) : ModuleName(ModuleName) {}
+  Identifier ModuleName;
+};
```

Comment:
What is the value of this class?  It just renames `Identifier` but then exposes it.

Reply:
Yes, it's no longer necessary.